active/InformationBar/InformationBar.md

```diff
@@ -398,6 +398,62 @@ Please view the guidance for
 for more information about text localization in your notification.
 
 
+# InfoBarPanel class
+
+A layout panel that positions its children horizontally if there’s available space,
+otherwise positions them vertically. This panel is intended to only be used as part
+of the ControlTemplate of the InfoBar control.
+
+## InfoBar Padding properties
+
+The `PaddingInHorizontalOrientation` property gets/sets the distance between the edges
+of the panel and its children, when the panel is laying out items horizontally.
+The `PaddingInVerticalOrientation` property does likewise when the panel is layout out
+items vertically.
+
+[API note: these padding properties are analogous to ex the 
+[StackPanel.Padding](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.StackPanel.Padding)
+property.]
+
+## InfoBar Spacing attached properties
+
+The `SpacingInHorizontalOrientation` attached property gets/sets the distance between i tems
+when the panel is laying out items in the horizontal orientation. Similarly 
+`SpacingInVerticalOrientation` for vertical layout. This property is set on the child items,
+so the spacing can be different between different item pairs.
+
+[API note: these spacing properties are analogous to ex the
+[StackLayout.Spacing](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.StackLayout.Spacing)
+property, except they're attached properties so that they can be set on the panel children.]
```

Comment:
Not really analogous.

StackLayout.Spacing is a scalar and is not additive. That spacing is the distance between adjacent elements,

```xml
<!-- Child1 and Child2 have 2 units of space between them -->
<StackLayout Spacing="2"><Child1/><Child2/></StackLayout>
```

In InfoBarPanel, the spacing is a Thickness and is additive, as noted in the next paragraph. If adjacent elements both have non-zero spacing along the shared edge, this value adds them together.

```xml
<!-- Child1 and Child2 have 2+2=4 units of space between them -->
<InfoBarPanel><Child1 IBP.SIHO="2"/><Child2 IBP.SIHO="2" /></StackLayout>
```

Thickness+Additive behavior appears to be what XAML calls "Margin".

Should these be named **Margin**InHorizontalOrientation / **Margin**InVerticalOrientation?

If we rename, then we also should decide which margin wins if the children have a Margin directly applied.

Reply:
Discussion: See earlier comment, agreed on making this "margin". This margin is additive with the existing Margin property.